config/environments/test.rb

```diff
@@ -14,7 +14,6 @@
   # https://guides.rubyonrails.org/configuring.html#config-cache-classes
   config.cache_classes = false
 
-  # See https://github.com/rails/rails/issues/40613#issuecomment-727283155
   config.action_view.cache_template_loading = false
```

Comment:
GitHub's diff viewer seems confused about the change, but I pulled the branch and confirmed that it is indeed setting `loading = false` here.